package.json

```diff
@@ -58,10 +58,12 @@
     "babel-preset-travix": "^1.1.0",
     "babel-register": "^6.16.3",
     "commander": "^2.9.0",
+    "create-react-class": "^15.5.2",
```

Comment:
@asci that is a peerDependency which means that is used by one of our dependencies, not our code. It's to solve this issue: https://github.com/JedWatson/react-input-autosize/issues/82